electrum/wallet.py

```diff
@@ -2426,6 +2427,16 @@ def __init__(self, db, storage, *, config):
         # for a few seconds!
         self.synchronize()
 
+        # create lightning keys
+        if self.can_have_lightning():
+            self.init_lightning()
+        ln_xprv = self.db.get('lightning_privkey2')
+        # lnworker can only be initialized once receiving addresses are available
+        # therefore we instantiate lnworker in DeterministicWallet
+        self.lnworker = LNWallet(self, ln_xprv) if ln_xprv else None
+        # does it make sense to instantiate lnbackups without lnworker?
```

Comment:
actually, yes: a wallet may have channel backups regardless of lnworker activation